### PR TITLE
Telegraf reorg

### PR DESCRIPTION
**Pull request by noramullen1**

### CODE REVIEW

File: content/telegraf/v1.22/troubleshoot.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Troubleshoot Telegraf
+description: Resolve common issues with Telegraf.
+menu:
+  telegraf_1_22_ref:
```
Comment (noramullen1) [inline content/telegraf/v1.22/troubleshoot.md:5]: Not sure if this belongs here in reference or elsewhere?

**Comment by noramullen1** [comment]
Reorg of 1.22 (release notes coming in separate PR)

**Review by sjwang90 — APPROVED** [review]
Nora + I reviewed yesterday and looks good

File: content/telegraf/v1.22/configure_plugins/external_plugins/_index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+[External plugins](/EXTERNAL_PLUGINS.md) are external programs that are built outside
+of Telegraf that can run through an `execd` plugin. These external plugins allow for
+more flexibility compared to internal Telegraf plugins. Benefits to using external plugins include:
+- Access to libraries not written in Go
+- Using licensed software (not available to open source community)
+- Including large dependencies that would otherwise bloat Telegraf
+- Using your external plugin immediately without waiting for the Telegraf team to publish
+- Easily convert plugins between internal and external using the [shim](/telegraf/latest/external_plugins/shim/)
```
Comment (kelseiv) [inline content/telegraf/v1.22/configure_plugins/external_plugins/_index.md]: broken link..did we drop shim? (did a search and can't find)

let's run a link check on this branch being so much has moved around
& do a search and replace `/telegraf/latest` with `/{{< latest "telegraf" >}}`

**Pull request merged**